package.json

```diff
@@ -36,6 +36,7 @@
   },
   "devDependencies": {
     "eslint": "^3.19.0",
-    "eslint-plugin-react": "^7.1.0"
+    "eslint-plugin-react": "^7.1.0",
+    "node_env": "production"
```

Comment:
`node_env` does not belong in `devDependencies`

Reply:
I forget where the docs said to put that, in another Dependencies? 
@dahlbyk

Reply:
does it go in SCRipts. @dahlbyk